src/TakeLast.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * @template T
+ * @template U
+ * @param {Array<T>} list
+ * @param {U} count
+ * @returns {Array<T>}
+ */
+function TakeLast(list, count) {
+  if (list.length < count) throw 'Array has not enough elements';
+
+  return list.slice(-count);
```

Comment:
From within your function use ` this`  when referring to the array instead of passing in `list`. The typing will also need to be fixed to reflect the change.

Reply:
Thanks for the comment! Think I have fixed it.